vite.sdk-assets-config.js

```diff
@@ -2,10 +2,29 @@ const path = require("path");
 const mergeOptions = require('merge-options');
 const commonOptions = require("./vite.common-config.js");
 
+const pathsToExport = [
+    "index.js",
+    "index.css",
+    "download-sandbox.html"
+];
+
 export default mergeOptions(commonOptions, {
     root: "src/",
     base: "./",
     build: {
         outDir: "../target/asset-build/",
+        rollupOptions: {
+            output: {
+              assetFileNames: (chunkInfo) => {
+                  // Get rid of the hash so we can consistently reference these
+                  // files in our `package.json` `exports`
+                  if(pathsToExport.includes(path.basename(chunkInfo.name))) {
+                    return "assets/[name].[ext]";
+                  }
+
+                return "assets/[name]-[hash][extname]";
+              }
```

Comment:
For reference, docs for the `rollupOptions.output.assetFileNames` option, https://rollupjs.org/guide/en/#outputassetfilenames